quactography/solver/qaoa_solver_qu_edge.py

```diff
@@ -0,0 +1,108 @@
+import multiprocessing
+import itertools
+from qiskit.primitives import Estimator, Sampler
+from qiskit.circuit.library import QAOAAnsatz
+from scipy.optimize import minimize
+import numpy as np
+
+from quactography.solver.io import save_optimization_results
+
+
+alpha_min_costs = []
+
+
+# Function to find the shortest path in a graph using QAOA algorithm with parallel processing:
+def find_longest_path(args):
```

Comment:
What takes time in the function? Is it the `minimze` part? If so, I wonder if all the stuff after the minimization should be outside of the parallelized function, since it does not need to be done in parallel... We usually save the output in the script itself, and I had never seen a file saved inside a multiprocess. 

But if it works well for you or if I am missing some reason behind this, keep it as is!